### PR TITLE
fix(schema): add missing `type` property where there was either `true` or `false` in an `enum`

### PR DESCRIPTION
**Pull request by ahl**

### CODE REVIEW

**Comment by G-Rath** [comment]
Could you provide some details about why you think we need to make this change? we test our schemas against a bunch of payload examples and they're passing fine as is using `ajv`.

**Comment by ahl** [comment]
I count 17 places where there's either `"enum": [true]` or `"enum": [false]`:

```
$ git ls-files | grep schema.json | xargs -L1 jq -c '.. | select(.enum? == [true])'
{"type":"boolean","enum":[true]}
{"type":"boolean","enum":[true]}
{"type":"boolean","enum":[true]}
{"enum":[true]}
{"type":"boolean","enum":[true]}
{"type":"boolean","enum":[true],"description":"Indicates whether or not the pull request is a draft."}
{"type":"boolean","enum":[true],"description":"Whether the release is identified as a prerelease or a full release."}
{"type":"boolean","enum":[true],"description":"Whether the repository is archived.","default":false}
{"type":"boolean","enum":[true],"description":"Whether the repository is private or public."}
$ git ls-files | grep schema.json | xargs -L1 jq -c '.. | select(.enum? == [false])'
{"type":"boolean","enum":[false]}
{"type":"boolean","enum":[false]}
{"type":"boolean","enum":[false]}
{"type":"boolean","enum":[false]}
{"type":"boolean","enum":[false]}
{"type":"boolean","enum":[false],"description":"Indicates whether or not the pull request is a draft."}
{"type":"boolean","enum":[false],"description":"Whether the repository is private or public."}
{"type":"boolean","enum":[false],"description":"Whether the repository is archived.","default":false}
```

This is the only instance that omits the type.

This PR was to add consistency, but if that's not a goal feel free to close it.

**Comment by wolfy1339** [comment]
With that explanation I understand what you are trying to fix.

The `"type": "boolean"` should be in the same object as the `enum`, not in the parent object

File: payload-schemas/api.github.com/fork/event.schema.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         { "$ref": "common/repository.schema.json" },
         {
           "type": "object",
-          "properties": { "fork": { "enum": [true] } },
+          "properties": { "type": "boolean", "fork": { "enum": [true] } },
```
Comment (wolfy1339) [inline payload-schemas/api.github.com/fork/event.schema.json:14]:
```suggestion
          "properties": { "fork": { "type": "boolean",  "enum": [true] } },
```

**Review by wolfy1339 — CHANGES_REQUESTED** [review]

**Review by wolfy1339 — APPROVED** [review]

**Comment by wolfy1339** [comment]
Can you lint the files?

**Pull request merged**